alacritty/src/input.rs

```diff
@@ -33,13 +34,23 @@ use alacritty_terminal::vi_mode::ViMotion;
 
 use crate::clipboard::Clipboard;
 use crate::config::{Action, Binding, Config, Key, ViAction};
-use crate::event::{ClickState, Mouse};
+use crate::event::{ClickState, Event, Mouse};
+use crate::scheduler::{Scheduler, SELECTION_SCROLLING_EVENT};
 use crate::url::{Url, Urls};
 use crate::window::Window;
 
 /// Font size change interval.
 pub const FONT_SIZE_STEP: f32 = 0.5;
 
+/// Interval for mouse scrolling during selection outside of the boundaries.
+const SELECTION_SCROLLING_INTERVAL: Duration = Duration::from_millis(25);
```

Comment:
Could we increase tick rate, otherwise it feel really slow compared to what it was before. Especially when in fullscreen or when you don't really have gaps on bottom.

Reply:
I think the current tick rate is a great minimum value, otherwise accurately selecting stuff becomes a hassle.

The primary goal isn't to move through the scrollback as quickly as possible.

Reply:
Ehm, I don't really like it because it's very slow and I think doing a bit faster is reasonable. I'd say  `12` ms is what it should be, so the selection will be a bit smoother as well, while the rate is still normal.

>accurately selecting stuff becomes a hassle.

Or you're waiting ages to scroll. You're fine to re select a bit, since you're not scrolling that fast with `12ms`, and `25ms` doesn't protect you from inaccuracy, since you always select something that was invisible a frame before, so the chance to reselect even with 25ms is high already.

Reply:
> Ehm, I don't really like it because it's very slow and I think doing a bit faster is reasonable. I'd say 12 ms is what it should be, so the selection will be a bit smoother as well, while the rate is still normal.

12 ms is far too quick for me, it's also higher than most monitor's refresh rate, so then the speed will depend on your monitor which isn't a great idea.

If your goal is to cover a huge distance as quickly as possible, use a feature that doesn't have the goal to still accurately preserve your mouse pointer position or make sure there's plenty of space around your window.

Reply:
I've slightly tightened up the values to 20 - 20. Maybe that's a good middle ground? Let me know what you think.